### PR TITLE
Bigquery-denormalized fix integration tests

### PR DESCRIPTION
**Pull request by suhomud**
## What
in order to fix integration test,  `spec.json` and `destination_definitions.yaml` should be consistent. 
old spec.json: 
```
  "supportsNormalization": false,
  "supportsDBT": true,  
```
old destination_definitions.yaml: 
```
- name: BigQuery (denormalized typed struct)
  destinationDefinitionId: 079d5540-f236-4294-ba7c-ade8fd918496
  dockerRepository: airbyte/destination-bigquery-denormalized
  dockerImageTag: 1.2.9
  documentationUrl: https://docs.airbyte.com/integrations/destinations/bigquery
  icon: bigquery.svg
  normalizationConfig:
    normalizationRepository: airbyte/normalization
    normalizationTag: 0.2.25
    normalizationIntegrationType: bigquery
  resourceRequirements:
    jobSpecific:
      - jobType: sync
        resourceRequirements:
          memory_limit: "1Gi"
          memory_request: "1Gi"
  releaseStage: beta

```
result: 
```
BigQueryDenormalizedGcsDestinationAcceptanceTest > specNormalizationValueShouldBeCorrect() FAILED
specNormalizationValueShouldBeCorrect 
```
new spec.json: 
```
  "supportsNormalization": false,
  "supportsDBT": false,
```
destination_definitions.yaml: 
```
- name: BigQuery (denormalized typed struct)
  destinationDefinitionId: 079d5540-f236-4294-ba7c-ade8fd918496
  dockerRepository: airbyte/destination-bigquery-denormalized
  dockerImageTag: 1.2.9
  documentationUrl: https://docs.airbyte.com/integrations/destinations/bigquery
  icon: bigquery.svg
  resourceRequirements:
    jobSpecific:
      - jobType: sync
        resourceRequirements:
          memory_limit: "1Gi"
          memory_request: "1Gi"
  releaseStage: beta
```
## How
*Describe the solution*

## Recommended reading order
1. `spec.json` 
2. `destination_definitions.yaml` 
3.  `BigQueryDenormalizedDestinationAcceptanceTest.java`

### CODE REVIEW

**Review by edgao — APPROVED** [review]
I think this is correct? but @pedroslopez can you confirm

also, isn't destination_definitions generated using the spec.json files? Shouldn't this have caused build failures, since `./gradlew build` would have generated a nonempty diff?

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml [hunk 1]
```diff
@@ -60,10 +60,6 @@
   dockerImageTag: 1.2.9
   documentationUrl: https://docs.airbyte.com/integrations/destinations/bigquery
   icon: bigquery.svg
-  normalizationConfig:
-    normalizationRepository: airbyte/normalization
-    normalizationTag: 0.2.25
-    normalizationIntegrationType: bigquery
```
Comment (pedroslopez) [inline airbyte-config/init/src/main/resources/seed/destination_definitions.yaml:66]: To confirm: are we saying here that this destination does not support normalization? (from the name `-denormalized` i would assume so, but confirming 😄 )

**Comment by pedroslopez** [comment]
> also, isn't destination_definitions generated using the spec.json files? Shouldn't this have caused build failures, since ./gradlew build would have generated a nonempty diff?

@edgao No, definitions are not generated. You may be thinking of destination_specs.yaml which _are_ generated by running a spec command on the connector and saving the results.

There's an upcoming ticket to remove supportsDbt and related fields from the spec and keep that information as part of the definitions only. I think this is it but there's not much info in it 😅  https://github.com/airbytehq/airbyte/issues/18239

**Review by pedroslopez — APPROVED** [review]

**Review by grishick — APPROVED** [review]

**Review by grishick — APPROVED** [review]

**Comment by octavia-approvington** [comment]
Approve this <br/> imagine the commander saying yes

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**